project2/screens/Search/ListItem.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { useNavigation } from "react-navigation-hooks";
+import { Text, TouchableOpacity, View } from "react-native";
+import s from "../../Styles";
+
+const SearchListItem = ({ id, title }) => {
+  const { navigate } = useNavigation();
+
+  return (
+    <TouchableOpacity onPress={() => navigate("Movie", { id })}>
+      <View style={s.row}>
+        <Text style={s.listText}>{title}</Text>
+      </View>
+    </TouchableOpacity>
+  );
+};
+
+export default SearchListItem;
```

Comment:
This woks fine just as a component that navigates onPress -- there shouldn't be a need for the extra render function. Note also the use of the `useNavigation` hook to grab the `navigate` function -- this is directly equivalent to passing it in via props.